client/src/component/EventDialog/CreateEventDialog.js

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from "react";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import {
+  makeStyles,
+} from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import FormLabel from "@material-ui/core/FormLabel";
+import Radio from "@material-ui/core/Radio";
+import RadioGroup from "@material-ui/core/RadioGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import FormControl from "@material-ui/core/FormControl";
+
+const useStyles = makeStyles((theme) => ({
+  form: {
+    textAlign: "center",
+    color: theme.palette.text.secondary,
+    "& > *": {
+      margin: 8,
+      width: "80ch",
+    },
+  },
+}));
+
+export default function MaxWidthDialog(props) {
+  const [state, setState] = useState({
+    user_id: "",
+    event_name: "",
+    type: localStorage.getItem("type"),
```

Comment:
this should be in state. if it needs to be persistent across multiple components, it can be a state in the parent component, and then passed down as a prop
https://reactjs.org/docs/thinking-in-react.html